pkg/policy/k8s/cilium_cidr_group.go

```diff
@@ -175,11 +182,15 @@ func getCIDRGroupRefs(cnp *types.SlimCNP) []string {
 				}
 			}
 		}
+		for _, ingress := range spec.IngressDeny {
+			for _, rule := range ingress.FromCIDRSet {
+				if len(rule.Cidr) == 0 {
+					cidrGroupRefs = append(cidrGroupRefs, string(rule.CIDRGroupRef))
+				}
+			}
+		}
 		for _, egress := range spec.Egress {
```

Comment:
Any reason we don't also check EgressDeny?

Reply:
Not really, just that https://github.com/cilium/cilium/pull/30624 was opened after this one.
I was thinking about doing that in a follow up PR, but let's do it here, it is easy enough. :+1: